infra/gravity/gravity_test.go

```diff
@@ -0,0 +1,49 @@
+package gravity
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+
+	sshutils "github.com/gravitational/robotest/lib/ssh"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGravityOutput(t *testing.T) {
+	for _, s := range []struct {
+		label string
+		fn    sshutils.OutputParseFn
```

Comment:
as discussed will be changed after implemented in gravity repo in next few days.